examples/cifar_cnntransformer/main.py

```diff
@@ -1,83 +1,88 @@
 """
 This example demonstrates the use of a CNN and a Transformer-Encoder
-for image classification on CIFAR10.
-
+for image classification on CIFAR10 Dataset, using PyTorch Lightning.
 Reference: See kale.embed.attention_cnn for more details.
```

Comment:
Some technical reference (e.g. a paper on cnn transformer) might be better than an API reference.